file-server/Dockerfile

```diff
@@ -1,24 +1,90 @@
 ARG TAG=dev
 
-FROM compdem/polis-client-admin:${TAG}          as admin
-FROM compdem/polis-client-participation:${TAG}  as participation
-FROM compdem/polis-client-report:${TAG}         as report
+# polis-client-admin
+# Gulp v3 stops us from upgrading beyond Node v11
+FROM docker.io/node:11.15.0-alpine
 
-FROM node:16.9.0-alpine
+WORKDIR /client-admin/app
 
-WORKDIR /app
+RUN apk add git --no-cache
 
-COPY package*.json ./
+COPY client-admin/package*.json ./
+RUN npm install
+
+COPY client-admin/polis.config.template.js polis.config.js
+# If polis.config.js exists on host, will override template here.
+COPY client-admin/. .
+
+ARG GIT_HASH
+RUN npm run deploy:prod
+
+# polis-client-participation
+# # Gulp v3 stops us from upgrading beyond Node v11
+FROM docker.io/node:11.15.0-alpine
+
+WORKDIR ../../client-participation/app
```

Comment:
This should probably be just `/client-participation/app`. Or even `/app`, since you end up getting a new image for each call to `FROM` IIUC.